src/pyfaf/utils/format.py

```diff
@@ -31,7 +33,7 @@ def as_table(headers, data, margin=1, separator=' '):
 
     widths = reduce(
         lambda x, y: map(
-            lambda a_b: max(a_b[0], a_b[1]), zip(x, y)
+            lambda a_b: max(a_b[0], a_b[1]), list(zip(x, y))
```

Comment:
I am confused - you imported zip from six.moves so the zip returns list, right? Why are you then calling list on the list?

Reply:
It returns iterator (you can see [https://www.eflglobal.com/version-agnostic-python-part-1/](https://www.eflglobal.com/version-agnostic-python-part-1/)).

Reply:
six.moves just make sure that the name of the function is the same. Which is proably not needed, because zip() has the same name in both python2 and python3 and it work for me even without the import. But true is that every documentation mention this import.